test/integration/src/test/java/io/pravega/test/integration/controller/server/ControllerServiceTest.java

```diff
@@ -163,7 +163,7 @@ public void testControllerService() throws Exception {
         //Different name in same scope
         createAStream(scope1, streamName2, controller, config3);
         createAStream(scope1, streamName3, controller, config3);
-
+        /*
```

Comment:
why have we commented this out? 
we should remove the code if its not to be used

Reply:
added by mistake. removed